exercises/fkt/sternchen.py

```diff
@@ -0,0 +1,22 @@
+height = input("Enter a height for the rectangle")
+width = input("Enter a width for the rectangle")
+y = int(height)
+x = int(width)
+def constructionOfBase(number):
+    for repetitions in range(number, 0, -1):
+        print("*", end="")
+    print("")
```

Comment:
Exemplarisch bei dieser Funktion, gilt aber überall:
1. Das Programm Funktioniert, top! 
2. Namensgebung ist eine Philosophie-Frage, die Python-Community verwendet `_` als Trennzeichen zwischen Wörtern aber bspw. JavaScript macht es mit Gross-Kleinbuchstaben.
3. Was ist der Vorteil in Ihrem Code von `range(number, 0, -1)` gegenüber `range(number)`? --> Wenn möglich die einfachste und am besten verständliche Version verwenden...

```suggestion
def construction_of_base(number):
    for repetitions in range(number):
        print("*", end="")
    print("")
```

Reply:
Anfangs dachte ich, dass man alles definieren mussten bei `range()`, deshalb hatte ich auch dies so geschrieben...
Und wegen der Bennenung, das werde ich für künftig beachten, ich wusste von der Vielfalt an Namensgebung für verscheidenen Programmiersprachen nicht.
Danke!